backends/tsdb/backend.go

```diff
@@ -111,6 +111,15 @@ func (b *Backend) newAdapter(session *frames.Session, password string, token str
 		return nil, err
 	}
 
+	if b.backendConfig.Workers == 0 {
+		resp, err := container.GetClusterMDSync(&v3io.GetClusterMDInput{})
```

Comment:
This will check the number of VNs on every write or delete.

Reply:
why? b.backendConfig.Workers won't be 0 on the second time here